### DISCUSSION TITLE
WAS 2301 update docs???

### DISCUSSION THREAD

**Discussion by bonzer2u** [microsoft/WSA#247] [General]
Greetings,
My WAS was auto updated 2 days ago via MS store to ver. 2301.40000.7.0 however I can find no change log or mention of this update.
Please advise, thx...

I meant WSA...... DOH!

**Comment by bansalsushant** [comment]
You can look at the release notes here: https://learn.microsoft.com/en-us/windows/android/wsa/release-notes

**Reply by Erkam246** [reply]
> You can look at the release notes here: https://learn.microsoft.com/en-us/windows/android/wsa/release-notes

I think you misunderstand something the problem here is that the Version: 2301.40000.7.0 is not in the Release Notes

**Reply by bonzer2u** [reply]
Correct, there is no update in documentation for version xxxx.7.0

**Reply by nieubank** [reply]
> Correct, there is no update in documentation for version xxxx.7.0

The .7 release includes a graphics update that fixes a crash that was affecting some apps on ARM64.

**Reply by bonzer2u** [reply]
Thx for the info....
